app/controllers/damages_controller.rb

```diff
@@ -30,8 +30,7 @@ def create
     end
   end
 
-  def edit
-  end
+  def edit; end
```

Comment:
Do you need an empty controller action?

Reply:
Forgot you mentioned that in another PR and that I confirmed it...